### PR TITLE
🙏 Update community link for v1.29.1

### PR DESCRIPTION
**Pull request by damianpumar**

### CODE REVIEW

File: docs/_source/_common/general.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 We are not perfect so we love to get feedback.\
 Do you feel anything is missing in our documentation or do you simply want some help? \
-Feel free to contact our favorite developer advocate David, via [email](mailto:david@argilla.io), [Slack](https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g), or by scheduling a video meeting in his [Calendly](https://calendly.com/argumentation-at-argilla/30min).
+Feel free to contact our favorite developer advocate David, via [email](mailto:david@argilla.io), [Slack](http://hf.co/join/discord), or by scheduling a video meeting in his [Calendly](https://calendly.com/argumentation-at-argilla/30min).
```
Comment (frascuchon) [inline docs/_source/_common/general.md]: ??
```suggestion
Feel free to contact our favorite developer advocate David, via [email](mailto:david@argilla.io), [Discord](http://hf.co/join/discord), or by scheduling a video meeting in his [Calendly](https://calendly.com/argumentation-at-argilla/30min).
```

**Comment by sdiazlor** [comment]
@frascuchon @damianpumar I believe there is a discontinuity regarding the docs, because the current ones are already updated for Discord, for instance here: https://docs.argilla.io/en/latest/community/contributing.html. But I think that in `main`, but the current docs in this PR are from 1.29 with old information.

**Comment by damianpumar** [comment]
> @frascuchon @damianpumar I believe there is a discontinuity regarding the docs, because the current ones are already updated for Discord, for instance here: https://docs.argilla.io/en/latest/community/contributing.html. But I think that in `main`, but the current docs in this PR are from 1.29 with old information.

Hi! but this version has other docs, I mean, the current one is for latest verrsion, not for this.

**Comment by sdiazlor** [comment]
> > @frascuchon @damianpumar I believe there is a discontinuity regarding the docs, because the current ones are already updated for Discord, for instance here: https://docs.argilla.io/en/latest/community/contributing.html. But I think that in `main`, but the current docs in this PR are from 1.29 with old information.
> 
> Hi! but this version has other docs, I mean, the current one is for latest verrsion, not for this.

I'm not sure if I get it right. I mean, the current latest docs in this domain are for 1.29 with changes in `main` after the official 1.29 release.  Just wanted to make sure that adding this 1.29.1 docs originated from the 1.29 without the posterior updates is not going to create discontinuity or miss these changes.

File: docs/_source/_common/next_steps.html [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
+      href="#Next-steps"
+      title="Permalink to this headline"
+      >#</a
+    >
+  </h2>
+  <p>If you want to continue learning Argilla:</p>
+  <p>
+    🙋‍♀️ Join the
+    <a class="reference external" href="http://hf.co/join/discord"
+      >Argilla Slack community</a
```
Comment (frascuchon) [inline docs/_source/_common/next_steps.html]:
```suggestion
      >Argilla Discord community</a
```

File: docs/_source/community/contributing.md [hunk 3]
```diff
@@ -7,45 +6,43 @@ Thank you for investing your time in contributing to the project! Any contributi
...
-* **GitHub**: It is a platform and cloud-based service that uses git and allows developers to collaborate on projects. To contribute to Argilla, you'll need to create an account. Check the [Contributor Workflow with Git and Github](#contributor-workflow-with-git-and-github) for more info.
-* **Developer Documentation**: To collaborate, you'll need to set up an efficient environment. Check the [developer documentation](/community/developer_docs.md) to know how to do it.
-* **Schedule a meeting with our developer advocate**: If you have more questions, do not hesitate to contact to our developer advocate and [schedule a meeting](https://calendly.com/argilla-office-hours/30min).
-
+- **Slack**: You are welcome to join the [Argilla Slack community](http://hf.co/join/discord), where you can keep in touch with other users, contributors and the Argilla team. In the following [section](#first-contact-in-slack), you can find more information on how to get started in Slack.
```
Comment (frascuchon) [inline docs/_source/community/contributing.md]:
```suggestion
- **Discord**: You are welcome to join the [Argilla Discord community](http://hf.co/join/discord), where you can keep in touch with other users, contributors and the Argilla team. In the following [section](#first-contact-in-slack), you can find more information on how to get started in Slack.
```

File: docs/_source/community/contributing.md [hunk 4]
```diff
@@ -7,45 +6,43 @@ Thank you for investing your time in contributing to the project! Any contributi
...
+- **GitHub**: It is a platform and cloud-based service that uses git and allows developers to collaborate on projects. To contribute to Argilla, you'll need to create an account. Check the [Contributor Workflow with Git and Github](#contributor-workflow-with-git-and-github) for more info.
+- **Developer Documentation**: To collaborate, you'll need to set up an efficient environment. Check the [developer documentation](/community/developer_docs.md) to know how to do it.
+- **Schedule a meeting with our developer advocate**: If you have more questions, do not hesitate to contact to our developer advocate and [schedule a meeting](https://calendly.com/argilla-office-hours/30min).
 
 ## First Contact in Slack
 
-Slack is a very useful tool for more casual conversations and to answer day-to-day questions. Click [here](https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g) to join our Slack community effortlessly.
+Slack is a very useful tool for more casual conversations and to answer day-to-day questions. Click [here](http://hf.co/join/discord) to join our Slack community effortlessly.
```
Comment (frascuchon) [inline docs/_source/community/contributing.md]: Discord?

File: docs/_source/community/contributing.md [hunk 5]
```diff
@@ -221,12 +212,11 @@ Additionally, we will include your changes in the next release based on our [dev
...
-
 ## Additional resources
 
 Here are some helpful resources for your reference.
 
-* [Configuring Slack](https://slack.com/help/articles/218080037-Getting-started-for-new-Slack-users), a guide to learn how to configure Slack.
-* [Pro Git](https://git-scm.com/book/en/v2), a book to learn Git.
-* [Git in VSCode](https://code.visualstudio.com/docs/sourcecontrol/overview), a guide to learn how to easily use Git in VSCode.
-* [GitHub Skills](https://skills.github.com/), an interactive course to learn GitHub.
+- [Configuring Slack](https://slack.com/help/articles/218080037-Getting-started-for-new-Slack-users), a guide to learn how to configure Slack.
```
Comment (frascuchon) [inline docs/_source/community/contributing.md]: discord

File: docs/_source/community/migration-rubrix.md [hunk 6]
```diff
@@ -63,9 +63,9 @@ This will allow you to work with previous Rubrix datasets from your new Argilla
 
 This migration switch can help you with a more gradual transition into Argilla.
 
->:warning: New datasets created from argilla won't be visible from the old rubrix instance.
+> :warning: New datasets created from argilla won't be visible from the old rubrix instance.
 
-> 🚒 **We'd love to support you with this migration process. The easiest way is to contact us through the [Slack Community](https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g) or to open a GitHub issue**
+> 🚒 **We'd love to support you with this migration process. The easiest way is to contact us through the [Slack Community](http://hf.co/join/discord) or to open a GitHub issue**
```
Comment (frascuchon) [inline docs/_source/community/migration-rubrix.md]:
```suggestion
> 🚒 **We'd love to support you with this migration process. The easiest way is to contact us through the [Discord Community](http://hf.co/join/discord) or to open a GitHub issue**
```

File: docs/_source/getting_started/installation/deployments/huggingface-spaces.md [hunk 7]
```diff
@@ -122,7 +123,7 @@ As a next step, you can check the [Argilla Tutorials](https://docs.argilla.io/en
 
 ## Feedback and support
 
-If you have improvement suggestions or need specific support, please join [Argilla Slack community](https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g) or reach out on [Argilla's GitHub repository](https://github.com/argilla-io/argilla).
+If you have improvement suggestions or need specific support, please join [Argilla Slack community](http://hf.co/join/discord) or reach out on [Argilla's GitHub repository](https://github.com/argilla-io/argilla).
```
Comment (frascuchon) [inline docs/_source/getting_started/installation/deployments/huggingface-spaces.md]:
```suggestion
If you have improvement suggestions or need specific support, please join [Argilla Discord community](http://hf.co/join/discord) or reach out on [Argilla's GitHub repository](https://github.com/argilla-io/argilla).
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 8]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: 🗯 Community Discussions
-    url: https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g
+    url: http://hf.co/join/discord
     about: Our Slack Community loves to discuss Argilla and NLP topics
```
Comment (sdiazlor) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    about: Our Discord Community loves to discuss Argilla and NLP topics
```

File: docs/migration-rubrix.md [hunk 9]
```diff
@@ -63,7 +62,6 @@ This will allow you to work with previous Rubrix datasets from your new Argilla
 
 This migration switch can help you with a more gradual transition into Argilla.
 
->:warning: New datasets created from argilla won't be visible from the old rubrix instance.
-
-> 🚒 **We'd love to support you with this migration process. The easiest way is to contact us through the [Slack Community](https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g) or to open a GitHub issue**
+> :warning: New datasets created from argilla won't be visible from the old rubrix instance.
 
+> 🚒 **We'd love to support you with this migration process. The easiest way is to contact us through the [Slack Community](http://hf.co/join/discord) or to open a GitHub issue**
```
Comment (frascuchon) [inline docs/migration-rubrix.md]:
```suggestion
> 🚒 **We'd love to support you with this migration process. The easiest way is to contact us through the [Discord Community](http://hf.co/join/discord) or to open a GitHub issue**
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -15,9 +15,8 @@ For contributions, we work together to regreen the earth with [JustDiggit](https
 
 ## Need some help?
 
-We understand that getting started might be scary and difficult, therefore, we provide a guided contributor mentorship program. Feel free to schedule a meeting [here](https://calendly.com/argilla-office-hours/30min) to get you started. Alternatively, you can start with [joining our Slack](https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g) and reaching out in our [contributor channel](https://rubrixworkspace.slack.com/archives/C05BCGM277B).
+We understand that getting started might be scary and difficult, therefore, we provide a guided contributor mentorship program. Feel free to schedule a meeting [here](https://calendly.com/argilla-office-hours/30min) to get you started. Alternatively, you can start with [joining our Slack](http://hf.co/join/discord).
```
Comment (sdiazlor) [inline CONTRIBUTING.md]:
```suggestion
We understand that getting started might be a bit difficult, therefore, you can join our discord channel [#argilla-general](https://discord.gg/hugging-face-879548962464493619) by selecting *Argilla* in *Channels and Roles* after joining the Discord.
```

File: README.md [hunk 11]
```diff
@@ -27,7 +26,7 @@
 <a href="https://www.linkedin.com/company/argilla-io">
 <img src="https://img.shields.io/badge/linkedin-blue?logo=linkedin"/>
 </a>
-<a href="https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g">
+<a href="http://hf.co/join/discord">
 <img src="https://img.shields.io/badge/slack-purple?logo=slack"/>
```
Comment (sdiazlor) [inline README.md]:
```suggestion
<img src="https://img.shields.io/badge/Discord-7289DA?&logo=discord&logoColor=white"/>
```

File: README.md [hunk 12]
```diff
@@ -58,7 +57,7 @@ We are an open-source community-driven project and we love to hear from you. Her
 
 - [Community Meetup](https://lu.ma/embed-checkout/evt-IQtRiSuXZCIW6FB): listen in or present during one of our bi-weekly events.
 
-- [Slack](https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g): get direct support from the community.
+- [Discord](http://hf.co/join/discord): get direct support from the community.
```
Comment (sdiazlor) [inline README.md]:
```suggestion
- [Discord](http://hf.co/join/discord): get direct support from the community in #argilla-general and #argilla-help.
```

File: argilla-frontend/README.md [hunk 13]
```diff
@@ -28,7 +28,7 @@
 https://github.com/argilla-io/argilla/assets/1107111/49e28d64-9799-4cac-be49-19dce0f6bd86
 
 <p align="center">
-<a  href="https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g">
+<a  href="http://hf.co/join/discord">
 <img src="https://img.shields.io/badge/JOIN US ON SLACK-4A154B?style=for-the-badge&logo=slack&logoColor=white" />
```
Comment (sdiazlor) [inline argilla-frontend/README.md]:
```suggestion
<img src="https://img.shields.io/badge/Discord-7289DA?&logo=discord&logoColor=white"/>
```

File: argilla-server/README.md [hunk 14]
```diff
@@ -27,7 +26,7 @@
 <a href="https://www.linkedin.com/company/argilla-io">
 <img src="https://img.shields.io/badge/linkedin-blue?logo=linkedin"/>
 </a>
-<a href="https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g">
+<a href="http://hf.co/join/discord">
 <img src="https://img.shields.io/badge/slack-purple?logo=slack"/>
```
Comment (sdiazlor) [inline argilla-server/README.md]:
```suggestion
<img src="https://img.shields.io/badge/Discord-7289DA?&logo=discord&logoColor=white"/>
```

File: argilla-server/docker/quickstart/README.md [hunk 15]
```diff
@@ -21,22 +21,19 @@
...
-
 ![imagen](https://user-images.githubusercontent.com/1107111/204772677-facee627-9b3b-43ca-8533-bbc9b4e2d0aa.png)
 
 <!-- https://user-images.githubusercontent.com/25269220/200496945-7efb11b8-19f3-4793-bb1d-d42132009cbb.mp4 -->
 
-
 <p align="center">
-<a  href="https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g">
+<a  href="http://hf.co/join/discord">
 <img src="https://img.shields.io/badge/JOIN US ON SLACK-4A154B?style=for-the-badge&logo=slack&logoColor=white" />
```
Comment (sdiazlor) [inline argilla-server/docker/quickstart/README.md]:
```suggestion
<img src="https://img.shields.io/badge/JOIN US ON DISCORD-7289DA?style=for-the-badge&logo=discord&logoColor=white" />
```

File: argilla/README.md [hunk 16]
```diff
@@ -27,7 +26,7 @@
 <a href="https://www.linkedin.com/company/argilla-io">
 <img src="https://img.shields.io/badge/linkedin-blue?logo=linkedin"/>
 </a>
-<a href="https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g">
+<a href="http://hf.co/join/discord">
 <img src="https://img.shields.io/badge/slack-purple?logo=slack"/>
```
Comment (sdiazlor) [inline argilla/README.md]:
```suggestion
<img src="https://img.shields.io/badge/Discord-7289DA?&logo=discord&logoColor=white"/>
```

File: argilla/README.md [hunk 17]
```diff
@@ -58,7 +57,7 @@ We are an open-source community-driven project and we love to hear from you. Her
 
 - [Community Meetup](https://lu.ma/embed-checkout/evt-IQtRiSuXZCIW6FB): listen in or present during one of our bi-weekly events.
 
-- [Slack](https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g): get direct support from the community.
+- [Discord](http://hf.co/join/discord): get direct support from the community.
```
Comment (sdiazlor) [inline argilla/README.md]:
```suggestion
- [Discord](http://hf.co/join/discord"): get direct support from the community in #argilla-general and #argilla-help.
```

File: docs/_source/community/contributing.md (hunk 3 above)
Comment (sdiazlor) [reply docs/_source/community/contributing.md]:
```suggestion
- **Discord**: You are welcome to join the [Argilla Discord community](http://hf.co/join/discord), where you can keep in touch with other users, contributors and the Argilla team. In the following [section](#first-contact-in-discord), you can find more information on how to get started in Discord.
```

File: docs/_source/community/contributing.md [hunk 18]
```diff
@@ -7,45 +6,43 @@ Thank you for investing your time in contributing to the project! Any contributi
...
+- **Git**: This is a very useful tool to keep track of the changes in your files. Using the command-line interface (CLI), you can make your contributions easily. For that, you need to have it [installed and updated](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) on your computer.
+- **GitHub**: It is a platform and cloud-based service that uses git and allows developers to collaborate on projects. To contribute to Argilla, you'll need to create an account. Check the [Contributor Workflow with Git and Github](#contributor-workflow-with-git-and-github) for more info.
+- **Developer Documentation**: To collaborate, you'll need to set up an efficient environment. Check the [developer documentation](/community/developer_docs.md) to know how to do it.
+- **Schedule a meeting with our developer advocate**: If you have more questions, do not hesitate to contact to our developer advocate and [schedule a meeting](https://calendly.com/argilla-office-hours/30min).
 
 ## First Contact in Slack
```
Comment (sdiazlor) [inline docs/_source/community/contributing.md]:
```suggestion
## First Contact in Discord
```

File: docs/_source/community/contributing.md (hunk 4 above)
Comment (sdiazlor) [inline docs/_source/community/contributing.md]:
```suggestion
Discord is a handy tool for more casual conversations and to answer day-to-day questions. As part of Hugging Face, we have set up some Argilla channels on the server. Click [here](http://hf.co/join/discord) to join the Hugging Face Discord community effortlessly.
```

File: docs/_source/community/contributing.md [hunk 19]
```diff
@@ -7,45 +6,43 @@ Thank you for investing your time in contributing to the project! Any contributi
...
 
-Slack is a very useful tool for more casual conversations and to answer day-to-day questions. Click [here](https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g) to join our Slack community effortlessly.
+Slack is a very useful tool for more casual conversations and to answer day-to-day questions. Click [here](http://hf.co/join/discord) to join our Slack community effortlessly.
 
 The following screens will be displayed, choose how you wish to join and enter the code sent to you via email.
```
Comment (sdiazlor) [inline docs/_source/community/contributing.md]:
```suggestion
When part of the Hugging Face Discord, you can select "Channels & roles" and select "Argilla" along with any of the other groups that are interesting to you. "Argilla" will cover anything about argilla and distilabel. You can join the following channels:
```

File: docs/_source/community/contributing.md [hunk 20]
```diff
@@ -7,45 +6,43 @@ Thank you for investing your time in contributing to the project! Any contributi
...
 
 ![argilla-slack](../_static/images/community/contributing/argilla-slack.png)
```
Comment (sdiazlor) [inline docs/_source/community/contributing.md]:
```suggestion
```

File: docs/_source/community/contributing.md [hunk 21]
```diff
@@ -7,45 +6,43 @@ Thank you for investing your time in contributing to the project! Any contributi
...
 
 Once you have joined the community, you'll be added to some channels by default, but below we show you all the community channels you can join:
```
Comment (sdiazlor) [inline docs/_source/community/contributing.md]:
```suggestion
```

File: docs/_source/community/contributing.md [hunk 22]
```diff
@@ -7,45 +6,43 @@ Thank you for investing your time in contributing to the project! Any contributi
...
 
-* **00-announcements**: 📣 Stay up-to-date on official Argilla
-* **01-introductions**:  👋 Say hi! to the community Fun facts are appreciated.
-* **02-support-and-questions**: 🙋‍♀️ Need help with Argilla or NLP? We are always here.
-* **03-discoveries-and-news**: 📚 Looking for resources and news related to everything NLP?
-* **04-contributors**: 🏗️ A channel for contributions and contributors.
-* **05-beta-testing**: 🚧 For access to the latest features and help us with testing the newest features.
-* **06-general**: 🐒 This channel is for... well, everything else.
-* **07-events-and-job-offers**: 👔 Would you like to share info about events, job offers or meetups?
+- **00-announcements**: 📣 Stay up-to-date on official Argilla
```
Comment (sdiazlor) [inline docs/_source/community/contributing.md]:
```suggestion
- **#argilla-general**: 📣 Stay up-to-date and general discussions.
- **#argilla-help**: 🙋‍♀️ Need assistance? We're always here to help. Select the appropriate label (argilla or distilabel) for your issue and post it.
```

File: docs/_source/community/contributing.md [hunk 23]
```diff
@@ -7,45 +6,43 @@ Thank you for investing your time in contributing to the project! Any contributi
...
+- **01-introductions**: 👋 Say hi! to the community Fun facts are appreciated.
+- **02-support-and-questions**: 🙋‍♀️ Need help with Argilla or NLP? We are always here.
+- **03-discoveries-and-news**: 📚 Looking for resources and news related to everything NLP?
+- **04-contributors**: 🏗️ A channel for contributions and contributors.
+- **05-beta-testing**: 🚧 For access to the latest features and help us with testing the newest features.
+- **06-general**: 🐒 This channel is for... well, everything else.
+- **07-events-and-job-offers**: 👔 Would you like to share info about events, job offers or meetups?
```
Comment (sdiazlor) [inline docs/_source/community/contributing.md]:
```suggestion
```

File: docs/_source/community/contributing.md (hunk 5 above)
Comment (sdiazlor) [inline docs/_source/community/contributing.md]:
```suggestion
- [Configuring Discord](https://support.discord.com/hc/en-us/categories/115000217151), a guide to learn how to get started with Discord.
```

File: docs/_source/community/migration-rubrix.md (hunk 6 above)
Comment (sdiazlor) [inline docs/_source/community/migration-rubrix.md]:
```suggestion
> 🚒 **We'd love to support you with this migration process. The easiest way is to contact us through the [Discord Community](http://hf.co/join/discord) or to open a GitHub issue**
```

File: docs/_source/getting_started/installation/deployments/huggingface-spaces.md (hunk 7 above)
Comment (sdiazlor) [inline docs/_source/getting_started/installation/deployments/huggingface-spaces.md]:
```suggestion
If you have improvement suggestions or need specific support, please join [Argilla Discord community](http://hf.co/join/discord) or reach out on [Argilla's GitHub repository](https://github.com/argilla-io/argilla).
```

File: docs/_source/index.rst [hunk 24]
```diff
@@ -62,7 +62,7 @@
    :maxdepth: 3
    :hidden:
 
-   Slack <https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g>
+   Slack <http://hf.co/join/discord>
```
Comment (sdiazlor) [inline docs/_source/index.rst]:
```suggestion
   Discord <http://hf.co/join/discord>
```

File: docs/migration-rubrix.md (hunk 9 above)
Comment (sdiazlor) [inline docs/migration-rubrix.md]:
```suggestion
> 🚒 **We'd love to support you with this migration process. The easiest way is to contact us through the [Discord Community](http://hf.co/join/discord) or to open a GitHub issue**
```

**Review by sdiazlor — COMMENTED** [review]
I just added most of the changes made here: https://github.com/argilla-io/argilla/pull/5101/files. At least to keep the coherence in general.

**Comment by damianpumar** [comment]
@sdiazlor @frascuchon 
I push just the frontend part.

Please take a look

**Review by frascuchon — APPROVED** [review]

**Pull request merged**